telemetry/opentelemetry-config/src/main/java/io/helidon/telemetry/otelconfig/HelidonOpenTelemetryServiceFactory.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2025 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.telemetry.otelconfig;
+
+import java.util.function.Supplier;
+
+import io.helidon.config.Config;
+import io.helidon.service.registry.Service;
+
+import io.opentelemetry.api.OpenTelemetry;
+
+@Service.Singleton
+@Service.RunLevel(Service.RunLevel.STARTUP)
```

Comment:
Do we really need a run level for this? 
Run level should be used if the component needs to open sockets, start processing repeating tasks etc. If the `OpenTelementry` instance may be needed by other services, it does not need run level, and it would be created when needed.